pyinstamation/bot.py

```diff
@@ -198,7 +198,7 @@ def follow_multiple_users(self, username_list, min_followers=None,
             )
 
     def unfollow(self, username):
-        if self._user_login:
+        if self.user_login:
```

Comment:
the login in `follow` and `unfollow` is not necessary. The login check it's already in higher abstractions.